src/catkin_pkg/package_templates.py

```diff
@@ -309,7 +311,7 @@ def _create_include_macro(package_template):
         if deplist:
             result += '\n# TODO: Check names of system library include directories (%s)' % ', '.join(deplist)
     if includes:
-        result += '\ninclude_directories(\n  %s\n)' % '\n  '.join(includes)
+        result += '\n%s' % '\n  '.join(includes)
```

Comment:
The logic doesn't indent the first item of the list? Maybe if we get of `results` all together (see the other comments) the lines 313-315 can be folded into this:

```
return '\n'.join(['  %s' % include for include in includes])
```